### ISSUE TITLE
Error when installing Gem: ld: library not found for -lpng

### ISSUE THREAD

**Issue by jsutherland1** [westonplatter/phashion#19]
I get this error when I run "gem install phashion"

```
ld: library not found for -lpng
```

I'm on OS X 10.8.4.

**Comment by ElliotCui** [comment]
I encountered the most same error when I was installing phashion.
```
Machine: MacBook Air (M1, 2020)
Os: macOS BigSur 11.2 (20D64)

$ gcc --version
Configured with: --prefix=/Library/Developer/CommandLineTools/usr --with-gxx-include-dir=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/c++/4.2.1
Apple clang version 12.0.0 (clang-1200.0.32.29)
Target: arm64-apple-darwin20.3.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin

Related:
I use homebrew, which has full support on NEW Mac(M1), to install pacakges.
I use rvm(1.29.12) to manage ruby(2.6.3 as default).
```
When I doing bundle install of my project with `Pashion` in my gemfile. I encountered troubleshooting: 
`ld: library not found for -ljpeg`.
Confirmed things are I have installed `jpeg, libjpeg, libpng` by following commands.
`brew install jpeg` and `brew install libjpeg`
`brew install libpng`

After lots of searching and tries, the solution is as follows:

1. install Phashion with config
`LIBRARY_PATH=$LIBRARY_PATH:/opt/homebrew/opt/libjpeg/lib gem install phashion -v '1.2.0'`
Then you will get troubleshooting: `fatal error: 'jpeglib.h' file not found`

2. last troubleshooting is because of clang path error. So
Add `export CPLUS_INCLUDE_PATH=/opt/homebrew/include` into `~/.zshrc`
Don't forget to `source ~/.zshrc`.

3. run `LIBRARY_PATH=$LIBRARY_PATH:/opt/homebrew/opt/libjpeg/lib gem install phashion -v '1.2.0'` once again.
Then You will get troubleshooting: `ld: library not found for -lpng`

4. run `LIBRARY_PATH=$LIBRARY_PATH:/opt/homebrew/opt/libjpeg/lib:/opt/homebrew/opt/libpng/lib gem install phashion -v '1.2.0'` once again.
Installation success!
